transport/src/main/java/io/netty5/channel/ChannelPipeline.java

```diff
@@ -541,4 +495,12 @@ default boolean isEmpty() {
 
     @Override
     ChannelPipeline flush();
+
+    /**
+     * The number of the outbound bytes that are buffered / queued in this {@link ChannelPipeline}. This number will
+     * affect the writability of the {@link Channel} together the buffered / queued bytes in the {@link Channel} itself.
+     *
+     * @return the number of buffered / queued bytes.
+     */
+    long pendingOutboundBytes();
```

Comment:
happy for better naming suggestions

Reply:
`outboundBytesQueued` or `queuedOutboundBytes`?

Reply:
I think the name is actually good enough ... Lets keep it as it is